src/module.d.ts

```diff
@@ -523,6 +523,7 @@ declare namespace posthog {
         request_batching?: boolean
         sanitize_properties?: (properties: posthog.Properties, event_name: string) => posthog.Properties
         properties_string_max_length?: number
+        mask_all_inputs?: boolean
```

Comment:
Please keep the docstrings in sync - there's one in module.d.ts and one in posthog-core explaining all of the options.